locales/jp.yml

```diff
@@ -1,11 +1,11 @@
 ---
 jp:
-  navigation:
+  global_navigation:
     support: サポート
-    contribute: Contribute
+    community: Community
     docs: Docs
     logo: Middleman
-    search_placeholder: ドキュメント検索
```

Comment:
I put up a fixup commit which updates the template to use this translation: 61f6c8c663afcdcee07ebbda9796f0b4f816f5a0